tests/cupy_tests/core_tests/test_ndarray_adv_indexing.py

```diff
@@ -538,29 +543,8 @@ def test_adv_setitem(self, xp, dtype):
     {'shape': (2, 3, 4), 'indexes': [[1]], 'value': 1},
     {'shape': (2, 3, 4), 'indexes': [[1, 0]], 'value': 1},
     {'shape': (2, 3, 4), 'indexes': [[1], [0]], 'value': 1},
-    {'shape': (2, 3, 4), 'indexes': [[1, 0], 2], 'value': 1},
-    {'shape': (2, 3, 4), 'indexes': [[1], slice(1, 2)], 'value': 1},
-    {'shape': (2, 3, 4), 'indexes': [[[1]], slice(1, 2)], 'value': 1},
 )
-@testing.gpu
-class TestArrayAdvancedIndexingSetitemScalarValueDeprecated:
-
-    @testing.for_all_dtypes()
-    @testing.numpy_cupy_array_equal()
-    def test_adv_setitem(self, xp, dtype):
-        a = xp.zeros(self.shape, dtype=dtype)
-        with testing.assert_warns(FutureWarning):
-            a[self.indexes] = self.value
-        return a
-
-
-@testing.parameterize(
-    {'shape': (0,), 'indexes': True, 'value': 1},
-    {'shape': (0,), 'indexes': (True,), 'value': 1},
-    {'shape': (0,), 'indexes': numpy.ones((), dtype=numpy.bool_), 'value': 1},
-    {'shape': (0,), 'indexes': numpy.zeros((), dtype=numpy.bool_), 'value': 1},
```

Comment:
These tests were split from `TestArrayAdvancedIndexingSetitemScalarValue` in https://github.com/cupy/cupy/pull/1503 for NumPy 1.13 support, so I've made them back there in this PR as numpy<1.13 is dropped.